spec/models/user_spec.rb

```diff
@@ -14,13 +14,10 @@
         expect { create(:user, :no_email) }.to raise_error(ActiveRecord::RecordInvalid)
       end
 
-      it 'are properly formatted' do
-        expect { create :user, :bad_email }.to raise_error(ActiveRecord::RecordInvalid)
-      end
-
-      it 'come from .gov' do
-        expect { create :user, :bad_email_domain }.to raise_error(ActiveRecord::RecordInvalid)
-      end
+      # TODO: Add a complex regex or use email verification
```

Comment:
Guessing there's no longer a requirement for a `.gov` email address?

Hopping into a meeting but didn't find the validation for that, either way.

Reply:
@jkassemi That's right. There is one external user who would not be using a `.gov` email and will use `ID.me` to login.
Modified the regex in devise.